0.preprocessing_data/scripts/0.patient_specific_preprocessing.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import argparse
+import pathlib
+import sys
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+argparse = argparse.ArgumentParser(
+    description="Copy files from one directory to another"
+)
+argparse.add_argument("--HPC", action="store_true", help="Type of compute to run on")
+
+# Parse arguments
+args = argparse.parse_args(args=sys.argv[1:] if "ipykernel" not in sys.argv[0] else [])
+HPC = args.HPC
+
+print(f"HPC: {HPC}")
+
+
+# In[3]:
+
+
+if HPC:
+    raw_image_dir_hpc = pathlib.Path("/pl/active/koala/GFF_Data/GFF-Raw/").resolve(
+        strict=True
+    )
+else:
+    # comment out depending on whose computer you are on
+    # mike's computer
+    raw_image_dir_local = pathlib.Path(
+        "/home/lippincm/Desktop/18TB/NF1_Patient_organoids/"
+    ).resolve(strict=True)
+    # Jenna's computer
+    # raw_image_dir_local = pathlib.Path("/media/18tbdrive/GFF_organoid_data/Cell Painting-NF0014 Thawed3-Pilot Drug Screening")
+
+
+# ## NF0014 specific preprocessing
+
```

Comment:
Recommend including what the renaming processing is that is being done for NF0014 as documentation. It isn't super obvious to me currently, other than it is renaming the parent directory in some way